.github/workflows/main.yml

```diff
@@ -135,13 +137,6 @@ jobs:
           - erlang: 21.x
             elixir: 1.6
 
-          - erlang: 20.3.8.23
-            elixir: 1.8
```

Comment:
I think it might surprise users that they must run OTP >= 21 with 1.6 <= Elixir <= 1.9. The [Compatibility between Elixir and Erlang/OTP ](https://hexdocs.pm/elixir/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp) claims that Elixir 1.6 through 1.9 support OTP 20.

Reply:
I can check again, which >otp20 API I used, I doubt it was important.

Reply:
Hah, whatever the actual problem here was, it's not there anymore. I've set up CI builds and tests for `erlang-cargo` as well for OTP 20-23 and they passed, so I removed this commit.

Reply:
:+1: